sdk/examples/wallet/offline_signing/2_sign_transaction.rs

```diff
@@ -44,7 +46,7 @@ async fn main() -> Result<()> {
     let protocol_parameters = ProtocolParameters::new(
         2,
         String::from("testnet"),
-        String::from("smr"),
+        Hrp::from_str_unchecked("smr"),
```

Comment:
Just `smr` should be fine, right? Since it's an example, I'd do the cleanest possible